tests/common/pom.xml

```diff
@@ -129,6 +129,23 @@
 			<artifactId>brave-tests</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>io.projectreactor.kafka</groupId>
+			<artifactId>reactor-kafka</artifactId>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>testcontainers</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
```

Comment:
all of the testcontainer ones can be `optional` cause only kafka tests will use `testcontainers`, right?